packages/cli/scripts/preinstall.js

```diff
@@ -6,7 +6,7 @@ export function main() {
   // `.slice(1)` removes `v` from `v16`
   const nodeMajorVersion = parseInt(nodeVersions[0].slice(1))
   const nodeMinorVersion = parseInt(nodeVersions[1])
-  if (nodeMajorVersion < 16 || nodeMajorVersion === 16 && nodeMinorVersion < 13) {
+  if (nodeMajorVersion < 16 || (nodeMajorVersion === 16 && nodeMinorVersion < 13)) {
```

Comment:
Ohoh, bug or just readability change?

Reply:
That's something that came from rebasing with `main`, let's find the original PR

Reply:
According to the JS parenthesisation rules, those parentheses can safely be dropped as they are redundant. 
Personally, I always strive for non-ambiguity of operator precedence via a "safe amount" of parentheses, so I would revert that specific change.

Reply:
It's just style here (prettier), parenthesis are nice to make it easier to read here.

Reply:
Ok, so not via merge but applying prettier here on code that somehow did not get it before.
Was a bit unexpected in a dependency PR.